lib/ansible/modules/system/user.py

```diff
@@ -420,8 +420,9 @@
 import time
 
 from ansible.module_utils import distro
-from ansible.module_utils._text import to_native, to_bytes, to_text
-from ansible.module_utils.basic import load_platform_subclass, AnsibleModule
+from ansible.module_utils.common.sys_info import get_platform_subclass
+from ansible.module_utils._text import to_native, to_bytes
```

Comment:
Alphabetize imports.
```suggestion
from ansible.module_utils._text import to_native, to_bytes
from ansible.module_utils.basic import AnsibleModule
from ansible.module_utils.common.sys_info import get_platform_subclass
```